api/src/main/kotlin/edu/wgu/osmt/task/Task.kt

```diff
@@ -40,7 +40,8 @@ import java.util.*
     JsonSubTypes.Type(value = RemoveCollectionSkillsTask::class, name = "RemoveCollectionSkillsTask"),
     JsonSubTypes.Type(value = ExportSkillsToCsvTaskV2::class, name = "ExportSkillsToCsvTaskV2"),
     JsonSubTypes.Type(value = RemoveItemTask::class, name = "RemoveCollectionSkillsTask"),
-    JsonSubTypes.Type(value = RemoveJobCodeTask::class, name = "RemoveJobCodeTask")
+    JsonSubTypes.Type(value = RemoveJobCodeTask::class, name = "RemoveJobCodeTask"),
```

Comment:
You don't need to add new task classes definition if you make use of the RemoveItem data class, you just need to provide the identifier as string and the apiResultPath to be forwarded to.